### ISSUE TITLE
MockUps not so responsive

### ISSUE THREAD

**Issue by KumarAbhirup** [gbougakov/JoinMyCall#1]
`assets/mockups/index.html` and `assets/mockups/guestStep1.html` aren't mobile friendly.

**Comment by KumarAbhirup** [comment]
`index.html` is now responsive.
The very minimalistic design of `index.html` looks odd on older smartphones with low resolution.

Maybe we add a logo at the top??

**Comment by gbougakov** [comment]
Sure!

**Comment by gbougakov** [comment]
`guestStep1.html` still looks odd. Really odd.

1. We should hide local view on mobile because it's not really important and does not fit
2. Add padding on `body` to make sure that text doesn't hit the edge of viewport

![IMG_1815](https://user-images.githubusercontent.com/11464592/58720791-d0764880-83db-11e9-88a5-8b07ac274da6.PNG)

**Comment by gbougakov** [comment]
And I think we need to change the fonts to the system ones:
```css
font-family: -apple-system, system-ui, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
```

**Comment by KumarAbhirup** [comment]
Okay, I just hid the local view pane in mobile. Tried all the methods provided, also added the `body` padding. Yet I think it looks odd.

I wasn't able to resize Remote View when Local View gets hidden, I'll pass that on to you.

What I know is, it is an easy fix somewhere here 👇
```
.grid {
  display: grid;
  grid-template-columns: 1fr 1fr;
  grid-auto-flow: row;
  grid-column-gap: 20px;
  grid-row-gap: 0px;
  width: 80%;
  max-width: 1000px;
  margin: 0px auto;
}

.grid article {
  width: 100%;
  padding: 15px;
}

.grid article img {
  width: 100%;
}
```

**Comment by KumarAbhirup** [comment]
> And I think we need to change the fonts to the system ones:
> 
> ```css
> font-family: -apple-system, system-ui, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
> ```

I used a custom font... should I replace it?

**Comment by KumarAbhirup** [comment]
Created the `development` branch. But, we'll edit the `html-mockups` branch for the changes you seek.

By tomorrow, we'll start the `vue.js` work.

**Comment by gbougakov** [comment]
Fixed in #3
